app/uk/gov/hmrc/partnershipidentificationfrontend/service/PartnershipIdentificationService.scala

```diff
@@ -52,7 +52,7 @@ class PartnershipIdentificationService @Inject()(connector: PartnershipIdentific
   def storeRegistrationStatus(journeyId: String,
                               registrationStatus: RegistrationStatus
                              )(implicit hc: HeaderCarrier): Future[SuccessfullyStored.type] =
-    connector.storeData[RegistrationStatus](journeyId, RegistrationKey, registrationStatus)
+    connector.storeData[RegistrationStatus](journeyId, RegistrationKey, registrationStatus)(RegistrationStatus.format, hc)
```

Comment:
Would it be better to specify the writes method of RegistrationStatus here as the connecter method signature is def storeData[DataType](journeyId: String, dataKey: String, data: DataType
                         )(implicit dataTypeWriter: Writes[DataType], hc: HeaderCarrier): Future[SuccessfullyStored.type]
?

Reply:
Sorry, I think that RegistrationStatus.format is the Writes method for registration status. It includes both reads and writes. I can't reference the method within that as it needs to be a class.

Reply:
I think you can use RegistrationStatus.format.writes, but it doesn't really matter as Scala seems to work out what is needed.